plugins/thunderbolt/fu-thunderbolt-retimer.c

```diff
@@ -97,6 +97,9 @@ fu_thunderbolt_retimer_setup(FuDevice *device, GError **error)
 	if (fu_device_get_vendor(device) == NULL)
 		fu_device_set_vendor(device, "Intel");
 
+	/* default unsigned */
+	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_UNSIGNED_PAYLOAD);
```

Comment:
this needs to go into _init() really -- setup is a bit too late as we want quirks to be able to override it.

Reply:
fixed in [c5a61a7](https://github.com/fwupd/fwupd/commit/c5a61a780ca0089ab482178a2292f660ec5bc429).